src/ducks/transactions/TransactionsPage.jsx

```diff
@@ -275,16 +287,20 @@ class TransactionsPage extends Component {
             period={this.state.currentMonth}
             onChange={this.handleChangeMonth}
           />
-          <TransactionTableHead />
+          {filteredTransactions.length > 0 ? (
+            !isMobile ? (
```

Comment:
`!isMobile && <TransactionTableHead />` pourrait fonctionner ?

Reply:
pareil https://github.com/cozy/cozy-banks/pull/101/files#diff-ceb283ed4feb6c879d56b055bed20371R273 et https://github.com/cozy/cozy-banks/pull/101/files#diff-ceb283ed4feb6c879d56b055bed20371R277

Reply:
fixed in https://github.com/cozy/cozy-banks/pull/101/commits/d651abc3b74a1c2455330d74991275a8f368f1b3